components/global/Header/Navigation.js

```diff
@@ -27,12 +28,17 @@ export default function Navigation({
 
   useClickEvent(handleClick);
 
+  useEffect(() => {
```

Comment:
can this effect be done at the same time as changing the `active` state? 

the current React recommendation is to do the state change + callback together, outside an effect: https://react.dev/learn/you-might-not-need-an-effect#notifying-parent-components-about-state-changes

Reply:
removed `useEffect` and instead (as sparingly as appears reasonable) call my parent state setter function where/when the active nav id is set.